config/hiveadmission/clusterdeployment-webhook.yaml

```diff
@@ -10,7 +10,7 @@ webhooks:
       # reach the webhook via the registered aggregated API
       namespace: default
       name: kubernetes
-      path: /apis/admission.hive.openshift.io/v1alpha1/clusterdeployments
+      path: /apis/admission.hive.openshift.io/v1alpha1/clusterdeploymentvalidators
```

Comment:
Can you please change `.metadata.name` and `.webhooks[].name` as well? These are on lines 5 and 7, respectively.

Reply:
Sure.